lib/id3c/cli/command/geocode.py

```diff
@@ -362,6 +369,7 @@ def us_street_lookup(address: dict) -> Lookup:
     SmartyStreets geocoding API is not present in the given *address* data.
     """
     lookup = Lookup()
+    print(address)
```

Comment:
Also printed to output file. Also, we probably just don't want to print addresses to the logs?

Reply:
Right, those were debugging statements for me that slipped in.